ament_copyright/ament_copyright/parser.py

```diff
@@ -54,14 +55,6 @@ def read(self):
     def parse(self):
         raise NotImplemented()
 
-    def identify_license(self, content, license_part):
-        for name, license in get_licenses().items():
-            if content is not None and getattr(license, license_part) == content:
-                self.license_identifier = name
-                break
-        else:
-            self.license_identifier = UNKNOWN_IDENTIFIER
-
```

Comment:
I am not sure what the override is needed for (since the Apache license has the same structure already) but I would certainly prefer the first or second option.